### DISCUSSION TITLE
how  run docker again after reboot ?

### DISCUSSION THREAD

**Discussion by tedihamdani** [mediagis/nominatim-docker#386] [General]
Hi
I have vps ubuntu + docker nominatim 4.1  (vmware)
all running good with no problem.
but today, accidentally.. I restart VPS.
right now I dont know how to run / start again docker
when "docker PS" I didnt see name of container and image
then I try to docker run -it e pbf url...
it said docker error respond from daemon : conflict....
I try docker start
it said "docker start requires at least 1 argument"

can you help me what I must to do? cause I am not so familiar with ubuntu and docker.
thanks before guys

**Comment by ashley-mort** [comment]
docker container ls -a

docker start CONTAINER_NAME
